packages/web/src/components/collection/CollectionCard.tsx

```diff
@@ -24,11 +28,8 @@ const messages = {
   hidden: 'Hidden'
 }
 
-type CardSize = 's' | 'm' | 'l'
-
-type CollectionCardProps = {
+type CollectionCardProps = Omit<CardProps, 'id'> & {
```

Comment:
does card props have like html id?

Reply:
yeah, im thinking maybe the move is to use "collectionId" and "userId" to not overload

Reply:
will address in the migration story